contracts/AnchoredView/SymbolConfiguration.sol

```diff
@@ -102,49 +111,123 @@ contract SymbolConfiguration {
      * @return The configuration metadata for the address
      */
     function getCTokenConfig(address cToken) public view returns(CTokenMetadata memory) {
-        return CTokenMetadata({
-                    openOracleKey: getOpenOracleKey(cToken),
-                    anchorOracleKey: getAnchorOracleKey(cToken),
-                    anchorAdditionalScale: getAdditionalScaleForAnchorPrice(cToken),
-                    underlyingPriceAdditionalScale: getAdditionalScaleForUnderlyingPrice(cToken),
-                    cTokenAddress: cToken
-                    });
-    }
+        if (cToken == cEthAddress) {
+            return CTokenMetadata({
+                        openOracleKey: "ETH",
+                        anchorOracleKey: address(0),
+                        baseUnit: 1e18,
+                        cTokenAddress: cEthAddress,
+                        priceSource: PriceSource.REPORTER,
+                        anchorSource: AnchorSource.FIXED_ETH,
+                        fixedReporterPrice: 0,
```

Comment:
I assume you are being forced to initialize these to 0, even though they aren't used?

Reply:
yes, no choice :(